README.build.md

```diff
@@ -19,9 +19,13 @@ export SDE_INSTALL=<path>
 ## Build
 
 ```bash
-./make-all.sh
+./make-all.sh [--target {tofino|dpdk}] [--prefix <prefix>]
```

Comment:
--sde-prefix?

Reply:
`--prefix` maps to `CMAKE_INSTALL_PREFIX`. It's the directory into which build artifacts will be installed.

`--sde-install` (formerly `--sde-prefix`) maps to `SDE_INSTALL_PREFIX`. It defaults to the value of the `SDE_INSTALL` environment variable.

Reply:
Can we specify this information in the readme, as just from prefix it's not clear

Reply:
Done.